### PR TITLE
Frontend Test Updates

### PR DESCRIPTION
**Pull request by chrisbendel**

### CODE REVIEW

File: frontend/src/screens/admin-home.tsx [hunk 1]
```diff
@@ -18,13 +18,13 @@ export default function AdminHome() {
...
-                <Route path="/rewards/" element={<AdminRewards />} />
+                <Route path="/" element={<Navigate to="/admin/banners" />} />
+                <Route path="/banners" element={<AdminBanners />} />
+                <Route path="/approve-studies" element={<ApproveStudies />} />
+                <Route path="/rewards" element={<AdminRewards />} />
                 <Route path="/workspaces/:studyId?" element={<AdminWorkspaces />} />
-                <Route path="/impersonate/" element={<Impersonation />} />
-                <Route path="/reports/" element={<AdminReports />} />
+                <Route path="/impersonate" element={<Impersonation />} />
+                <Route path="/reports" element={<AdminReports />} />
```
Comment (nathanstitt) [inline frontend/src/screens/admin-home.tsx:27]: as I recall, if the url lacks a trailing slash, the app will redirect to have a slash.  adding it at the beginning saves a roundtrip.  Can can test on staging to see if you set that there?

**Review by nathanstitt — APPROVED** [review]
a few small things, can you test the trailing slash thing please?

File: frontend/specs/admin.spec.ts [hunk 2]
```diff
@@ -1,64 +1,68 @@
...
+
+    await goToPage({ page: adminPage, path: '/admin' })
+    // await adminPage.waitForLoadState('networkidle')
+    // await adminPage.waitForURL('**/admin/banners')
+    await adminPage.waitForFunction(() => document.location.pathname == '/admin/banners')
+
+    await goToPage({ page: researcherPage, path: '/admin' })
+    // await researcherPage.waitForLoadState('networkidle')
+    // await researcherPage.waitForURL('**/studies/**')
+    await researcherPage.waitForFunction(() => document.location.pathname === '/studies')
```
[the remark this answers is not in the document]
Comment (chrisbendel) [reply frontend/specs/admin.spec.ts:12]: Yeah started using `waitForUrl` in this PR and just missed a couple spots! thanks for pointing this out

File: frontend/src/screens/admin-home.tsx (hunk 1 above)
Comment (chrisbendel) [reply frontend/src/screens/admin-home.tsx:27]: I logged into staging as an admin and hit both `/banners` and `/banners/` and it seems to have been functionally equivalent 🤔

**Pull request merged**